api/api-member/src/main/java/com/pgms/apimember/service/MemberService.java

```diff
@@ -21,9 +25,21 @@
 @RequiredArgsConstructor
 public class MemberService {
 
+	// TODO: role의 기본값을 어떻게 할 지 논의해봐야함
+	private static final String DEFAULT_ROLE_NAME = "ROLE_USER";
+
 	private final MemberRepository memberRepository;
+	private final RoleRepository roleRepository;
 	private final PasswordEncoder passwordEncoder;
 
+	public Long signUp(MemberSignUpRequest requestDto) {
+		if (memberRepository.existsByEmail(requestDto.email())) {
+			throw new MemberException(DUPLICATED_MEMBER_EMAIL);
+		}
+		validateNewPassword(requestDto.password(), requestDto.passwordConfirm());
+		return memberRepository.save(requestDto.toEntity(passwordEncoder, getRole(DEFAULT_ROLE_NAME))).getId();
```

Comment:
MemberSignUpRequest 내용과 같습니다! (dto와 service의 책임 분리)

```suggestion
		String encodedPassword = passwordEncoder.encode(requestDto.password());
		return memberRepository.save(requestDto.toEntity(encodedPassword, new Role("name"))).getId();

```